src/Toolkit/Toolkit.iOS/UI/Controls/PopupViewer/PopupViewer.iOS.cs

```diff
@@ -243,7 +245,7 @@ private void Refresh()
             if (!string.IsNullOrWhiteSpace(PopupManager.CustomDescriptionHtml))
             {
                 _customHtmlDescription.Hidden = false;
-                _customHtmlDescription.Text = PopupManager.CustomDescriptionHtml.ToPlainText();
+                _customHtmlDescription.Text = PopupManager.CustomDescriptionHtml?.ToPlainText();
```

Comment:
Doesn't IsNullOrWhiteSpace above already take care of this null-check?

Reply:
I wish, but ios annotation for IsNullOrWhiteSpace is missing. We can add `!` instead